deploy-agent/tests/unit/deploy/server/test_agent.py

```diff
@@ -60,37 +60,37 @@ def setUpClass(cls):
 
         cls.deploy_goal1 = {}
         cls.deploy_goal1['deployId'] = '123'
-        cls.deploy_goal1['envName'] = 'abc'
+        cls.deploy_goal1['envName'] = 'beacon'
```

Comment:
can we revert the name of env back?

Reply:
if I don't change it to beacon which is a real container name, the test fails since I added some container check in the function

Reply:
the envName is the container name

Reply:
have you considered mocking the container check in the test?